compiler/src/iree/compiler/Codegen/LLVMCPU/KernelDispatch.cpp

```diff
@@ -815,7 +836,7 @@ static int getRegisterSpaceBitsIfKnown(IREE::HAL::ExecutableTargetAttr target) {
 // `op` can simultaneously be allocated in SIMD registers. Does nothing when
 // SIMD register space can't be determined as a compile-time constant (e.g. Arm
 // SVE).
-static void limitVectorTileSizes(linalg::LinalgOp op,
+static void limitVectorTileSizes(Operation *op,
```

Comment:
Given this generalization, would it make sense to also remove the [`LinalgOp` condition](https://github.com/iree-org/iree/blob/8e3ddd6e03bd86f03d99b4d45f10c73276f0d91d/compiler/src/iree/compiler/Codegen/LLVMCPU/KernelDispatch.cpp#L2678) within `setRootConfig(mlir::FunctionOpInterface, TilingInterface)`?

Reply:
Probably. I don't want too much in one go :)